### PR TITLE
Added more command checks in CheckDHCTLDependencies method

### PR DESCRIPTION
**Pull request by ViktorKram**
## Description
Added more command checks in CheckDHCTLDependencies method to prevent unexpected fails in bootstrap.sh script.

## Why do we need it, and what problem does it solve?
Fix https://github.com/deckhouse/deckhouse/issues/5066. It solves the problem of unexpected bootstrap.sh failure, if some of shell-commands are missing.

## What is the expected result?
If some of shell-commands using in bootstrap.sh isn't installed, we'll get an error of a missing shell-command.

## Checklist
- [ ] The code is covered by unit tests.
- [x] e2e tests passed.
- [x] Documentation updated according to the changes.
- [ ] Changes were tested in the Kubernetes cluster manually.

## Changelog entries

```changes
section: bashible
type: fix
summary: client will get an error of missing shell-command in bootstrap.sh 
impact_level: low
```

### CODE REVIEW

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @ViktorKram. I've started the workflow for label 'e2e/run/openstack'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @ViktorKram. I've started the workflow for label 'e2e/run/openstack'...

  :fast_forward: `e2e: OpenStack, Containerd, Kubernetes 1.25` for `deckhouse:viktor.kramarenko/issue-5066/checking-all-utilities-are-installed` [started](https://github.com/deckhouse/deckhouse/actions/runs/6225314205).

**Review by z9r5 — DISMISSED** [review]

**Comment by z9r5** [comment]
/e2e/abort openstack;Standard;containerd;1.25 6225314205 6225314205-1-con-1-25 /sys/deckhouse-oss/install:pr5864 debian@188.124.39.178

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've recognized your '/e2e/abort' command and started the workflow...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've recognized your '/e2e/abort' command and started the workflow...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @ViktorKram. I've started the workflow for label 'e2e/run/openstack'...

**Comment by name212** [comment]
/e2e/abort openstack;Standard;containerd;1.25 6225957028 6225957028-1-con-1-25 /sys/deckhouse-oss/install:pr5864 debian@188.124.39.178

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @name212. I've recognized your '/e2e/abort' command and started the workflow...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @ViktorKram. I've started the workflow for label 'e2e/run/openstack'...

**Comment by z9r5** [comment]
/e2e/abort openstack;Standard;containerd;1.25 6225957028 6225957028-1-con-1-25 /sys/deckhouse-oss/install:pr5864 debian@188.124.39.178

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've recognized your '/e2e/abort' command and started the workflow...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @ViktorKram. I've started the workflow for label 'e2e/run/static'...

**Comment by ViktorKram** [comment]
/e2e/abort static;Static;containerd;1.25 6234842314 6234842314-1-con-1-25 /sys/deckhouse-oss/install:pr5864 astra@192.168.199.159

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @ViktorKram. I've recognized your '/e2e/abort' command and started the workflow...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @ViktorKram. I've started the workflow for label 'e2e/run/static'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @ViktorKram. I've started the workflow for label 'e2e/run/openstack'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @ViktorKram. I've started the workflow for label 'e2e/run/static'...

**Comment by z9r5** [comment]
/e2e/abort static;Static;containerd;1.25 6232682225 6232682225-1-con-1-25 /sys/deckhouse-oss/install:pr5864

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've recognized your '/e2e/abort' command and started the workflow...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've recognized your '/e2e/abort' command and started the workflow...

File: dhctl/pkg/operations/bootstrap.go [hunk 1]
```diff
@@ -188,9 +188,15 @@ func RunBashiblePipeline(sshClient *ssh.Client, cfg *config.MetaConfig, nodeIP,
...
 
-func CheckDHCTLDependencies(sshClient *ssh.Client) error {
+func CheckDHCTLDependencies(sshClient *ssh.Client, cfg *config.MetaConfig) error {
 	return log.Process("bootstrap", "Check DHCTL Dependencies", func() error {
-		var dependencyArgs = []string{"sudo", "rm", "tar"}
+		dependencyArgs := make([]string, 0, 13)
+		dependencyArgs = append(dependencyArgs, "sudo", "rm", "tar", "mount", "awk", "grep", "cut", "sed", "shopt",
+			"mkdir", "cp", "join")
+
+		if cfg.ClusterType == config.CloudClusterType {
```
Comment (name212) [inline dhctl/pkg/operations/bootstrap.go:197]: Why?

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @ViktorKram. I've started the workflow for label 'e2e/run/openstack'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @ViktorKram. I've started the workflow for label 'e2e/run/static'...

**Review by name212 — APPROVED** [review]

**Pull request merged**